client/src/Header.js

```diff
@@ -56,11 +56,6 @@ function Header() {
             Resources
           </Link>
         </li>
-        <li className={classNames('_item', { active })}>
-          <Link onClick={() => setActive(false)} to="/questions">
-            Questions
-          </Link>
-        </li>
```

Comment:
I'm removing Questions from the header here... instead, we'll access the questions for a given questionnaire in the QuestionnairesList...